MvvmCross-Forms/MvvmCross.Forms.Droid/Views/MvxFormsAndroidViewPresenter.cs

```diff
@@ -58,17 +58,18 @@ public virtual IMvxFormsPagePresenter FormsPagePresenter
 
         public override void Show(MvxViewModelRequest request)
         {
-            var action = GetPresentationAttributeAction(request.ViewModelType, out MvxBasePresentationAttribute attribute);
+            FormsPagePresenter.Show(request);
+            //var action = GetPresentationAttributeAction(request.ViewModelType, out MvxBasePresentationAttribute attribute);
 
-            if (FormsApplication.MainPage == null && attribute is MvxPagePresentationAttribute)
-            {
-                if(CurrentActivity is MvxFormsAppCompatActivity appCompatActivity)
-                    appCompatActivity.InitializeForms(null);
-                else if(CurrentActivity is MvxFormsApplicationActivity activity)
-                    activity.InitializeForms(null);
-            }
+            //if (FormsApplication.MainPage == null && attribute is MvxPagePresentationAttribute)
```

Comment:
Shouldn't this code be called somewhere? I think it is to startup without a splashscreen.

Reply:
It may have been originally but I think this is actually duplicating the way the setup logic now works and I suspect (yet to confirm) that it conflicts with it. Either way I don't believe this is the correct place to ensure XF is setup correctly